garak/resources/misp_descriptions.tsv

```diff
@@ -59,8 +59,8 @@ avid-effect:performance:P0404	Environmental safety	Safety from environmental har
 quality:Behavioral:ContentSafety:Toxicity	Toxicity	Toxicity
 quality:Security:PromptStability	Prompt Stability	Prompt Stability
 quality:Robustness:GenerativeMisinformation	Generative Misinformation	Generative Misinformation
-quality:Behavioral:ContentSafety:HateHarassment	Hate and harassment	Hate and harrassment
+quality:Behavioral:ContentSafety:HateHarassment	Hate and harassment	Hate and harassment
 quality:Behavioral:ContentSafety:Profanity	Profanity	Profanity
-quality:Security:Confidentiailty	Confidentiailty	Confidentiailty
+quality:Security:Confidentiality	Confidentiality	Confidentiality
 quality:Behavioral:ContentSafety:HarmfulViolent	Harmful or violent	Harmful or violent
-quality:Behavioral:ContentSafety:Sexualized	Sexualised	Sexualised
```

Comment:
Again, UK/US spelling

Reply:
Consistency as alternate spelling is used in in the same entry.

Reply:
I only know English English and haven't put learning variants on my roadmap - go with whatever works for you :)